src/applications/personalization/appointments/actions/index.js

```diff
@@ -211,10 +245,13 @@ export function fetchConfirmedFutureAppointments() {
 
         accumulator.push({
           additionalInfo: getAdditionalInfo(appointment),
-          id: appointment?.id,
+          id: appointment.id,
           isVideo: false,
-          providerName: appointment?.attributes?.providerPractice,
-          startsAt: startDate.toISOString(),
+          providerName: appointment.attributes?.providerPractice,
+          startsAt: getLocalAppointmentDate(
```

Comment:
This is the second major change. We are properly converting the CC (community care) appointment's date before storing it in `startsAt`.